### PR TITLE
[POAE7-2352] IN(varchar) support

### PR DESCRIPTION
**Pull request by yma11**
### What changes were proposed in this pull request?
in varchar array support

### Why are the changes needed?
support in varchar semantics for tpc-h

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
UT

### CODE REVIEW

File: cider/tests/utils/CiderTestBase.cpp [hunk 1]
```diff
@@ -37,6 +37,9 @@ void CiderTestBase::assertQuery(const std::string& sql,
   auto cider_input = json_file.size() ? json_file : sql;
   auto cider_res_batch = std::make_shared<CiderBatch>(
       ciderQueryRunner_.runQueryOneBatch(cider_input, input_[0]));
+
+  std::cout << cider_res_batch->toValueString() << std::endl;
```
Comment (jikunshang) [inline cider/tests/utils/CiderTestBase.cpp:41]: please remove

**Review by jikunshang — COMMENTED** [review]
LGTM.

File: cider/tests/utils/CiderTestBase.cpp (hunk 1 above)
Comment (yma11) [reply cider/tests/utils/CiderTestBase.cpp:41]: There is an UT failure I can't reproduce locally so add these lines for some output. Will remove once resolved.

**Comment by yma11** [comment]
@jikunshang @winningsix @chaojun-zhang please take a review.

File: cider/tests/utils/CiderTestBase.cpp [hunk 2]
```diff
@@ -37,6 +37,7 @@ void CiderTestBase::assertQuery(const std::string& sql,
   auto cider_input = json_file.size() ? json_file : sql;
   auto cider_res_batch = std::make_shared<CiderBatch>(
       ciderQueryRunner_.runQueryOneBatch(cider_input, input_[0]));
+
```
Comment (jikunshang) [inline cider/tests/utils/CiderTestBase.cpp:40]: revert?

**Review by jikunshang — APPROVED** [review]

**Review by winningsix — APPROVED** [review]

**Pull request merged**